### PR TITLE
Generate reports for front_end

### PR DESCRIPTION
**Pull request by jkim10**

### CODE REVIEW

File: .github/workflows/build_BE.yml [hunk 1]
```diff
@@ -90,8 +90,10 @@ jobs:
       - name: Creating Lint + Bug Reports
         run: |
           rm -f ../*_report.txt
-          flake8 ./*/*.py --show-source --statistics --verbose --max-line-length=100 --ignore=E --output-file=../lint_report.txt
-          flake8 ./*/*.py --show-source --statistics --verbose --max-line-length=100 --ignore=W --output-file=../bug_report.txt
+          flake8 ./*/*.py --show-source --statistics --verbose --max-line-length=100 --ignore=E --output-file=../BE_lint_report.txt
+          flake8 ./*/*.py --show-source --statistics --verbose --max-line-length=100 --ignore=W --output-file=../BE_bug_report.txt
+          npx eslint . -f node_modules/eslint-html-reporter/reporter.js -o ../FE_bug_report.html
+          npx standard | npx standard-reporter --json > ../FE_lint_report.json
```
Comment (ceguiluzrosas) [inline .github/workflows/build_BE.yml:96]: @jkim10 The error you're getting is related to `working_directory` which is defaulted to `backend` in line 10

**Review by ceguiluzrosas — CHANGES_REQUESTED** [review]
Comment below

File: .github/workflows/build_BE.yml [hunk 2]
```diff
@@ -87,19 +87,29 @@ jobs:
...
         shell: bash
-
+     
+      - name: Creating Lint + Bug Reports - Front-End
+        working-directory: ./frontend
+        run: |
+          rm -f ../FE_bug_report.html
+          rm -f ../FE_lint_report.json
+          npx eslint . -f node_modules/eslint-html-reporter/reporter.js -o ../FE_bug_report.html
+          npx standard | npx standard-reporter --json > ../FE_lint_report.json
```
Comment (ceguiluzrosas) [inline .github/workflows/build_BE.yml:103]: Strange...for some reason eslint isn't working for me in development.

Comment (jkim10) [reply .github/workflows/build_BE.yml:103]: yea not workign for me either. Going to redo the installation myself so i know the context

**Comment by jkim10** [comment]
@ceguiluzrosas think we're good now

**Comment by ceguiluzrosas** [comment]
> @ceguiluzrosas think we're good now

@jkim10 I know ❤️

**Pull request merged**